Setup/shield-version.txt

```diff
@@ -21,3 +21,4 @@ node-installer:latest node-installer:180214-12.26
 shield-logspout:latest shield-logspout:171123-17.23-642
 netdata:latest netdata:180114-08.17-1026
 speedtest:latest speedtest:180227-14.10-1464
+shield-autoupdate:latest shield-autoupdate:180314-15.28-1532
```

Comment:
Le't not change the prod version for now
(it will cause autoupdate)

Reply:
OK